README.md

```diff
@@ -1,31 +1,175 @@
-# schema-ui
+# reproschema-ui
 
 [![Build Status](https://travis-ci.org/ReproNim/schema-ui.svg?branch=master)](https://travis-ci.org/ReproNim/schema-ui)
 
+This repository contains the code for the user-interface for the [ReproSchema](https://github.com/ReproNim/reproschema).
 
-## Build Setup
+See it in action [HERE](https://schema.repronim.org/ui/)
 
+<!-- TOC -->
+
+- [reproschema-ui](#reproschema-ui)
+  - [List of supported input types](#list-of-supported-input-types)
+    - [Text inputs](#text-inputs)
+    - [Numerical values](#numerical-values)
+    - [Audio inputs](#audio-inputs)
+    - [Time and dates inputs](#time-and-dates-inputs)
+    - [Multiple options inputs](#multiple-options-inputs)
+    - [Others](#others)
+  - [App architecture](#app-architecture)
+  - [Reference material](#reference-material)
+  - [Serve the app on your computer](#serve-the-app-on-your-computer)
+    - [Install the dependencies](#install-the-dependencies)
+    - [Serve the app locally](#serve-the-app-locally)
+    - [Build for production with minification](#build-for-production-with-minification)
+    - [Build for production and view the bundle analyzer report](#build-for-production-and-view-the-bundle-analyzer-report)
+    - [Run unit tests](#run-unit-tests)
+    - [Run e2e tests](#run-e2e-tests)
+    - [Run all tests](#run-all-tests)
+
+<!-- /TOC -->
+
+## List of supported input types
+
+The `inputType` is checked by `src/components/InputSelector/InputSelector.vue` which then calls the appropriate scrip from those listed in the folder `src/components/Inputs/`.
+
+### Text inputs
+
+-   text: `WebTextInput/TextInput.vue`
+-   multitext: `MultiTextInput/MultiTextInput.vue`
+-   email: `EmailInput/EmailInput.vue`
+
+### Numerical values
+
+-   number: `WebIntegerInput/IntegerInput.vue`
+-   float:  `WebFloatInput/FloatInput.vue`
+
+### Audio inputs
+
+-   audioCheck: `AudioCheck/AudioCheck.vue`
+-   audioRecord: `WebAudioRecord/Audio.vue`
+-   audioPassageRecord: `WebAudioRecord/Audio.vue`
+-   audioImageRecord: `WebAudioRecord/Audio.vue`
+-   audioRecordNumberTask: `WebAudioRecord/Audio.vue`
+-   audioAutoRecord: `AudioCheckRecord/AudioCheckRecord.vue`
+
+### Time and dates inputs
+
+-   date: `YearInput/YearInput.vue`
+-   year: `YearInput/YearInput.vue`
+-   timeRange: `TimeRange/TimeRange.vue`
+
+### Multiple options inputs
+
+-   radio: `WebRadio/Radio.vue`
+-   slider: `SliderInput/SliderInput.vue`
+-   select: `SelectInput/SelectInput.vue`
+-   selectLanguage: `SelectInput/SelectInput.vue`
+-   selectCountry: `SelectInput/SelectInput.vue`
+-   selectState: `SelectInput/SelectInput.vue`
+
+### Others
+
+-   documentUpload: `DocumentUpload/DocumentUpload.vue`
+
+<!-- MultiPart
```

Comment:
I am not sure if those should be listed in the README.

Reply:
if they aren't listed here, where else?  these should be listed somewhere clearly visible I think so that people know what types are supported by reproschema-ui.
Also, do you think there is a better way to just use one type for `select` instead of having variations of it like `selectCountry` etc?

Reply:
also, `slider` is not a multiple options input.

Reply:
> Also, do you think there is a better way to just use one type for `select` instead of having variations of it like `selectCountry` etc?

No I think it is OK to keep it that way as long we make it clear in the doc :
- that all those input types use the same underlying component 
- how the list of choices to select from can be passed to that component.

I think those 3 types of "select" are fairly obvious ones to have around without committing to support a new type of "select" for each new use case.

Reply:
> also, `slider` is not a multiple options input.
Will update